app/models/volunteer.rb

```diff
@@ -0,0 +1,25 @@
+class Volunteer < FrozenRecord::Base
+  self.backend = FrozenRecord::Backends::Json
+
+  add_index :id, unique: true
+
+  def picture_path
+    @picture_path ||= if picture.blank?
+      "covidliste.png"
+    else
+      "volunteers/" + picture
+    end
+  end
+
+  def display_name
+    @display_name ||= if anon
+      "Bénévole anonyme"
+    elsif firstname || lastname
+      firstname + " " + lastname
```

Comment:
Si pour une raison ou pour une autre `firstname` ou `lastname` est `nil`, ça va péter.
Je te propose d'utiliser l'interpolation plutôt que la concaténation ici : 
```suggestion
      "#{firstname lastname}"
```

Reply:
`"#{firstname} #{lastname}"` plutôt non ?

Reply:
Euh oui pardon 🤦